docs/index.rst

```diff
@@ -58,6 +58,7 @@ User Documentation
    overview
    install
    getting_started
+   auto_examples/index
```

Comment:
If you want to put it here, it should only add one entry to the TOC called 'Tutorials' or 'Gallery of Tutorials'. At the moment it adds two obscure ``astropy.coordinates`` and ``astropy.io`` entries.